onnx_tf/handlers/backend/gather.py

```diff
@@ -3,12 +3,24 @@
 from onnx_tf.handlers.backend_handler import BackendHandler
 from onnx_tf.handlers.handler import onnx_op
 from onnx_tf.handlers.handler import tf_func
+from .gather_and_scatter_mixin import GatherAndScatterMixin
 
 
 @onnx_op("Gather")
 @tf_func(tf.gather)
-class Gather(BackendHandler):
+class Gather(GatherAndScatterMixin, BackendHandler):
 
   @classmethod
   def version_1(cls, node, **kwargs):
     return [cls.make_tensor_from_onnx_node(node, **kwargs)]
+
+  @classmethod
+  def version_11(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    indices = kwargs["tensor_dict"][node.inputs[1]]
+    axis = node.attrs.get("axis", 0)
+    result = cls.chk_idx_out_of_bounds_along_axis(x, axis, indices)
+    msg = 'Gather indices are out of bounds, please double check the indices and retry.'
+    with tf.control_dependencies([tf.compat.v1.assert_equal(result, True, message=msg)]):
+      indices = cls.process_neg_idx_along_axis(x, axis, indices)
+      return [tf.gather(x, indices, axis=axis, name=cls.tf_node_name(node.name))]
```

Comment:
@winston-zillow don't need to set name here. By default it will use the node.name as name

Reply:
@winnietsang are your sure that the `node.name` would be used automatically? How is the node name passed to the tensorflow op? it seems only the ops using `make_tensor_from_onnx_node` would inherit the node name.

Reply:
@winston-zillow, you are right node.name will not be use because it call tf.gather directly here, lets change it to use cls.make_tensor_from_onnx_node, then the name will be set for you automatically.

Reply:
@winnietsang done. Note: this problem exists for other ops that do not use the `make_tensor_from_onnx_node`